pkg/cache/tas_flavor_snapshot_test.go

```diff
@@ -30,22 +30,22 @@ func TestFreeCapacityPerDomain(t *testing.T) {
 	snapshot := &TASFlavorSnapshot{
 		leaves: leafDomainByID{
 			"domain2": &leafDomain{
-				freeCapacity: resources.Requests{
+				freeCapacity: resources.Resources{
 					corev1.ResourceCPU:    1000,
 					corev1.ResourceMemory: 2 * 1024 * 1024 * 1024, // 2 GiB
 				},
-				tasUsage: resources.Requests{
+				tasUsage: resources.Resources{
 					corev1.ResourceMemory: 1 * 1024 * 1024 * 1024, // 1 GiB
 					corev1.ResourceCPU:    500,
 				},
 			},
 			"domain1": &leafDomain{
-				freeCapacity: resources.Requests{
+				freeCapacity: resources.Resources{
 					corev1.ResourceMemory: 4 * 1024 * 1024 * 1024, // 4 GiB
 					corev1.ResourceCPU:    2000,
 					"nvidia.com/gpu":      1,
 				},
-				tasUsage: resources.Requests{
+				tasUsage: resources.Resources{
```

Comment:
@mimowo I guess that this is pretty weird since this represents `Resources of resources`.
So, I'm wondering if we can close this PR, then consider another better name...
WDYT?

Reply:
TBH, maybe it is not too bad. Don't close yet, I will re-think it post release and post KubeCon. I think definitely there is room for improvement, but I need to book time to think about it.

Reply:
SGTM, one possible way might change the package name.
Anyway, let's postpone this to later